### PR TITLE
Add vagrant capability for jenkins_master.

### PR DESCRIPTION
**Pull request by benpatterson**
I've used this locally for some Jenkins configuration work/testing.

Would appreciate any suggestions you might have on how to improve it.

### CODE REVIEW

**Comment by benpatterson** [comment]
@hkim823 this is running on ansible 1.6.1

@feanil I've changed the logic around the motd.d files like we discussed. (Did a push -f though, so our discussion was overwritten).

I had glanced at the jenkins_worker through the same process, and there are ec2-specific pieces in there, so the vagrant.stat could be useful on other files.

Thoughts/feedback? Thanks for the look.

File: playbooks/roles/common/tasks/main.yml [hunk 1]
```diff
@@ -15,6 +15,12 @@
     - "{{ COMMON_BIN_DIR }}"
     - "{{ COMMON_CFG_DIR }}"
 
+# Determine if machine is provisioned via vagrant
+# Some EC2-specific steps would need to be skipped
+- name: check if instance is vagrant
+  stat: path=/home/vagrant
+  register: vagrant_home_dir
+
```
Comment (hkim823) [inline playbooks/roles/common/tasks/main.yml:23]: I'm not sure doing it by version of Ubuntu spun up is a good way to do it, but doing a register / stat.exists works, but I think it should be done based on the items specifically listed.

File: vagrant/base/jenkins-standalone/requirements.txt [hunk 2]
```diff
@@ -0,0 +1,4 @@
+# Assumes vagrant is installed globally
+
+ansible==1.6.1
+boto==2.28.0
```
Comment (hkim823) [inline vagrant/base/jenkins-standalone/requirements.txt:4]: We have a global configuration requirements.txt file on the base level of the repo, which pins the versions of ansible and boto.

**Comment by feanil** [comment]
@benpatterson I've got something like this that I've been using for a while just opened it up as a PR(https://github.com/edx/configuration/pull/1140)  it does the more general case of what's happening here allowing you to run any role on a vagrant image.  What do you think about closing this and incorporating the MOTD changes into that PR?

**Comment by benpatterson** [comment]
Sure @feanil that sounds good to me. I'd personally wanted to eventually have something generic enough to run various different roles, and that takes us there :)

**Pull request closed without merging**